pythonforandroid/bootstraps/common/build/jni/application/src/start.c

```diff
@@ -182,18 +181,6 @@ int main(int argc, char *argv[]) {
    * replace sys.path with our path
    */
   PyRun_SimpleString("import sys, posix\n");
-  if (dir_exists("lib")) {
```

Comment:
Because those paths were the location for the old python, now we use the same packaging system as python3 (the paths are changed  and we zip some files...), so I think that those lines are useless but maybe I missed something ...Do you have some reason to maintain those lines?